### DISCUSSION TITLE
Actions

### DISCUSSION THREAD

**Discussion by StanMathers** [flet-dev/flet#75] [Q&A] [answered]
Hello, I'm so impressed and excited by Flet framework. I started using it yesterday and liked that, so I decided to rewrite one of my old projects which was built using Flask as a backend and Bootstrap as the front.

The website was simply a movie website, where you can register, discover movies, comment on a movie, and after clicking one of the movies you can see details as well as a trailer of it. For details in Flask, I have a URL that handles this process and the structure is like this "/details/<int: pk>".

The issue I had when rewriting this project using Flet is that I could not find anything in docs about changing URLs in the browser, just like "action" in HTML, and can anyone help me with that?

**Comment by FeodorFitsner** [comment, marked as answer]
Oh, it's so exciting you liked Flet! :) What you mean by actions is "Navigation and Routing" and this is something we are working on right now. Expect a new Flet release with routing support next week! There is an issue you can subscribe on to get a notification when it's done: https://github.com/flet-dev/flet/issues/76
